doc/how-to/use-external-builder-vm.md

```diff
@@ -0,0 +1,17 @@
+# How to use external builder vm
+
+This guide will cover how to use an external OpenStack builder VM to build and snapshot images.
+
+### Command
+
+Run the following command to create a snapshot image of a VM launched by the 
+github-runner-image-builder.
+
+To find out what flavor is available, run `openstack list flavor`.
+To find out what network is available, run `openstack list network`.
+
```

Comment:
Nitpick: I think the "To find out" sentences should be moved after the command block